### PR TITLE
Add `Build a native executable` guide

### PR DESCRIPTION
**Pull request by TharmiganK**
## Checklist

- [x] **Page addition**
  - [x] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+   ```
+
+2. Replace the content of the file service.bal :
+   ```ballerina
+   import ballerina/http;
+
+   listener http:Listener httpListener = new (8080);
+   service / on httpListener {
+       resource function get greeting() returns string { 
+       return "Hello, World!";
```
Comment (shafreenAnfar) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:89]: formatting issue.

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+
+2. Replace the content of the file service.bal :
+   ```ballerina
+   import ballerina/http;
+
+   listener http:Listener helloEP = new(9090);
+
+   service /helloWorld on helloEP {
+       resource function get sayHello() returns string {   
+       return "Hello, Docker!";
```
Comment (shafreenAnfar) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:199]: formatting issue.

**Review by sm1990 — APPROVED** [review]

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md (hunk 1 above)
Comment (TharmiganK) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:89]:
```suggestion
              return "Hello, World!"; 
```

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md (hunk 2 above)
Comment (TharmiganK) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:199]:
```suggestion
              return "Hello, Docker!";   
```

Comment (TharmiganK) [reply swan-lake/learn-the-platform/native-support/build-a-native-executable.md]: Fixed via [77afc98](https://github.com/ballerina-platform/ballerina-dev-website/pull/5837/commits/77afc98b47cef23ee18766c3860cc08782baa4c7)

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md (hunk 1 above)
Comment (TharmiganK) [reply swan-lake/learn-the-platform/native-support/build-a-native-executable.md]: Fixed via [77afc98](https://github.com/ballerina-platform/ballerina-dev-website/pull/5837/commits/77afc98b47cef23ee18766c3860cc08782baa4c7)

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**